modules/unsupported/css/src/test/java/org/geotools/styling/css/TranslatorSyntheticTest.java

```diff
@@ -127,6 +127,26 @@ public void fillBasic() throws IOException {
         assertNull(fill.getGraphicFill());
     }
 
+    @Test
+    public void testTranslateWithAutoNames() {
+        // Define the CSS string
+        String css = "@autoRuleNames 'true';\n" + "* { fill: #FF0000; }";
```

Comment:
I've changed the code to actually test what a user would write (the actual directive syntax). 
If the builds are green, I'll merge.